all.js

```diff
@@ -56,4 +63,75 @@ function signupCheck(){
     }else{
         return true;
     }
-}
\ No newline at end of file
+}
+
+// 傳送註冊API
+function signup() {
+    axios.post(`${apiURL}/users`, {
+        "user": {
+            "email": signupInfo.email,
+            "nickname": signupInfo.name,
+            "password": signupInfo.psw
+          }
+    })
```

Comment:
axios.post 的 payload（你要傳送的資料）傳入一般 js object 即可
```suggestion
    axios.post(`${apiURL}/users`, {
        user: {
            email: signupInfo.email,
            nickname: signupInfo.name,
            password: signupInfo.psw
          }
    })
```